src/uart.toit

```diff
@@ -99,6 +97,10 @@ class UartPixelStrip extends UartEncodingPixelStrip_:
   If your strip is RGB (24 bits per pixel), leave $bytes_per_pixel at
     3.  For RGB+WW (warm white) strips with 32 bits per pixel, specify
     $bytes_per_pixel as 4.
+  Note: You must update the whole strip.  If your strip has 15 pixels
```

Comment:
You could consider making this a `# Note` section, or so. (But I'm fine this way).